frontend/src/scenes/insights/Insights.tsx

```diff
@@ -56,6 +56,51 @@ function InsightHotkey({ hotkey }: { hotkey: HotKeys }): JSX.Element {
     return !isMobile() ? <span className="hotkey">{hotkey}</span> : <></>
 }
 
+function ComputationTimeAndRefresh({
+    lastRefresh,
+    loadResults,
+}: {
+    lastRefresh: string
+    loadResults: (refresh: boolean) => void
+}): JSX.Element {
+    const [, setRerenderCounter] = useState(0)
+
+    useEffect(() => {
+        const rerenderInterval = setInterval(() => {
+            setRerenderCounter((previousValue) => previousValue + 1)
+        }, 30000)
+        return () => {
+            clearInterval(rerenderInterval)
+        }
+    })
```

Comment:
Do you want this to run once on mount? If so, you might need a [] dependency here